.github/workflows/wpt-test.yml

```diff
@@ -31,57 +29,31 @@ jobs:
           WPT_API_KEY: ${{ secrets.HA_API_KEY }}
 
       - name: Run WebPageTest for more websites
-        run: |
-          # Get the PR body
-          PR_BODY="$(cat <<'EOF'
-          ${{ github.event.pull_request.body }}
-          EOF
-          )"
-
-          # Read PR body into an array, removing line breaks and carriage returns
-          declare -a lines
-          while IFS= read -r line; do
-            lines+=("${line//[$'\r\n']}")
-          done <<< "$PR_BODY"
-
-          # Find the index of the line after "**Test websites**:"
-          start_index=-1
-          for ((i=0; i<${#lines[@]}; i++)); do
-            if [[ "${lines[$i]}" == *"**Test websites**:"* ]]; then
-              start_index=$((i + 1))
-              break
-            fi
-          done
-
-          # If the index is valid, then parse the URLs
-          if [ $start_index -gt -1 ]; then
-            # Initialize an array for URLs
-            declare -a URLS
-            url_pattern="((http|https|ftp):\/\/[a-zA-Z0-9.-]+(\.[a-zA-Z]{2,4})(\/[a-zA-Z0-9_.-]+)*(\/?)(\?[a-zA-Z0-9_.-]+=[a-zA-Z0-9%_.-]+)*(\#?)([a-zA-Z0-9%_.-=]+)*)"
+        run: node tests/wpt.js
+        env:
+          WPT_SERVER: "webpagetest.httparchive.org"
+          WPT_API_KEY: ${{ secrets.HA_API_KEY }}
+          PR_BODY: ${{ github.event.pull_request.body }}
```

Comment:
What's weird about [this test run](https://github.com/HTTPArchive/custom-metrics/actions/runs/10047940354/job/27803652669?pr=137) is that it's using the original PR description before it was edited. There was a ~15 hour gap between the edit and the test run, so I'm not sure why it's not picking up the test websites that were added. Any ideas?

Reply:
I see you triggered to re-run the previously run action here - it uses the same inputs as the initial attempt. I'm not 100% sure, but that's what I always experienced before.

The easiest way here would be to do a new commit on the PR branch and check the newly triggered run.

Reply:
Ok thanks, makes sense